main.md

```diff
@@ -802,7 +802,7 @@ allows for the release of individual members of the address claim separately.
 ```json
 {
   "iss": "https://example.com/issuer",
-  "sub_jwk": {
+  "cnf": {
```

Comment:
There's a layer of nesting missing here per https://datatracker.ietf.org/doc/html/rfc7800#section-3.2 that uses "jwk" as the confirmation method name.  i.e.:

```
"cnf":{
    "jwk":{
        "kty": "RSA",
          ...  n and e ... 
 }}```